src/main/java/courierpackage/CourierGenerator.java

```diff
@@ -0,0 +1,46 @@
+package courierpackage;
+
+public class CourierGenerator {
+    public static Courier randomCourier(){
+        return new Courier()
+                .withLogin("fa1235")
+                .withPassword("fa2315")
+                .withFirstName("fa2141");
+    }
+    public static Courier randomCourierWithoutLogin(){
+        return new Courier()
+                .withLogin(null)
+                .withPassword("fa231");
+    }
+
+    public static Courier randomCourierWithoutPassword(){
+        return new Courier()
+                .withLogin("fa231")
+                .withPassword(null);
```

Comment:
вот тут тест loginCourierWithoutPassword как раз из-за использования null завершается с ошибкой 
java.lang.IllegalStateException: Cannot invoke the path method because no content-type was present in the response and no default parser has been set.